core/field.ts

```diff
@@ -110,7 +113,28 @@ export abstract class Field<T = any>
    * field is not yet initialized. Is *not* guaranteed to be accurate.
    */
   private tooltip: Tooltip.TipInfo | null = null;
-  protected size_: Size;
+
+  /** This field's dimensions. */
+  private size: Size = new Size(0, 0);
```

Comment:
Style guide suggests something like `sizeInternal`:
```suggestion
  private sizeInternal: Size = new Size(0, 0);
```

Reply:
I am confused about this, why? it's already `private`, why would we also add the word `internal`

Reply:
Quoting [the Styleguide section on get and set accesors says](go/tsjs-style#classes-getters-and-setters):
> If an accessor is used to hide a class property, the hidden property _may_ be prefixed or suffixed with any whole word, like `internal` or `wrapped`.

They do not provide a rational, but I suspect the reason is to avoid clashing with the name of the accessors.  In this case the accessors are named `size_` (which, with the trailing underscore, is contrary to current naming policy) but if it were `size` then the internal property would need to be named something else.